spacialgaze-plugins/misc.js

```diff
@@ -372,6 +372,15 @@ exports.commands = {
 			msg += '<button class="button" name="send" value="/emote add, ' + target[1] + ', ' + target[2] + '">Add emote</button></center>';
 			delete user.tokens[target[0]];
 			return SG.messageSeniorStaff(msg);
+		case 'disableintroscroll':
+			if (!target[2]) return this.errorReply('/usetoken disableintroscroll, [room]');
+			let roomid = toId(target[1]);
+			if (Rooms[roomid]) return this.errorReply(`${roomid} is not a room.`);
```

Comment:
You are also trying to access `Rooms` as if it were an object; rather, it's a function. It should be `if (!Rooms(roomid))`, like it is later in your code.